crypto/asn1/x_algor.c

```diff
@@ -110,13 +110,17 @@ int X509_ALGOR_copy(X509_ALGOR *dest, const X509_ALGOR *src)
         if ((dest->algorithm = OBJ_dup(src->algorithm)) == NULL)
 	    return 0;
 
-    if (src->parameter)
+    if (src->parameter) {
```

Comment:
The old code is present in master - so if it is wrong then it needs to be addressed there also.

Reply:
Will update on Master.